scripts/startup-script.sh

```diff
@@ -9,23 +9,25 @@ apt-get -y install python-dev python-pip
 sudo pip install -U pip
 sudo pip install virtualenv
 
-MANAGER_IP=$(ip -4 addr show dev eth0 | grep -E '^[[:space:]]*inet' | xargs | awk '{print $2}' | cut -d/ -f1)
-SSH_KEY=$HOME/.ssh/key
-INPUTS=$HOME/inputs.yaml
-BLUEPRINT=$HOME/simple-manager-blueprint.yaml
 USER=default
+MANAGER_IP=$(ip -4 addr show dev eth0 | grep -E '^[[:space:]]*inet' | xargs | awk '{print $2}' | cut -d/ -f1)
```

Comment:
Info: on Google Cloud instances (at least currently) may have only one natwork interface and routing/firewall is based on tags, so both public and private addresses will be the same as `eth0` interface's address.